core/include/moveit/task_constructor/stages/predicate_filter.h

```diff
@@ -65,6 +65,7 @@ class PredicateFilter : public WrapperBase
 	void onNewSolution(const SolutionBase& s) override;
 
 	void setPredicate(const Predicate& p) { setProperty("predicate", p); }
+	void setIgnoreFilter(bool ignore) { setProperty("ignore_filter", ignore); }
```

Comment:
Please add a description for this property. It's semantics is not clear to me yet.
Why would you like to ignore the filter at all? If so, you can simply remove the wrapper stage, can't you?

Reply:
it's just simpler to set the property than change the structure of the task if you optionally want to filter, e.g., by additional path constraints.

Reply:
OK. A comment would be nice anyway.

Reply:
I already added a property description.